tests/io_test.py

```diff
@@ -62,16 +62,58 @@ def test_can_crop_waveform():
     assert isinstance(wav, Tensor)
     assert sr == 16000
 
+# File Like Object Tests
+def test_can_load_from_file_like():
+    "Load entire wav of file like"
+    loader = Audio()
+
+    with open("tests/data/dev00.wav", "rb") as f:
+        wav, sr = loader(f)
+
+    assert isinstance(wav, Tensor)
+    assert sr == 16000
+
 
-def test_crops_are_correct_shape():
-    sr = 160001
-    secs = random.randint(5, 11)
-    waveform = torch.randn(1, secs * sr)
+def test_can_crop_from_file_like():
+    "Load cropped sections from file like objects"
     loader = Audio()
-    shape = None
-    for segment in SlidingWindow(end=secs):
-        wav, sr = loader.crop({"waveform": waveform, "sample_rate": sr}, segment)
-        if shape is None:
-            shape = wav.shape
-        else:
-            assert shape == wav.shape
+
+    with open("tests/data/dev00.wav", "rb") as f:
+        segment = Segment(0.2, 0.7)
+        wav, sr = loader.crop(f, segment)
+
+    assert isinstance(wav, Tensor)
+    assert sr == 16000
+    assert wav.shape[1] == 0.5 * 16000
+
+def test_can_crop_from_download_like():
+    "Load cropped sections from file like objects"
+    loader = Audio()
+
+    with open("tests/data/dev00.wav", "rb") as f:
+        segment = Segment(0.2, 0.7)
+        wav, sr = loader.crop(f, segment)
+
+    assert isinstance(wav, Tensor)
+    assert sr == 16000
+    assert wav.shape[1] == 0.5 * 16000
+
+def test_can_load_from_download():
```

Comment:
This test will need to be updated to use the same syntax as torchaudio 0.8 release note.